prowler/providers/aws/services/ec2/ec2_launch_template_no_secrets/ec2_launch_template_no_secrets.metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+  "Provider": "aws",
+  "CheckID": "ec2_launch_template_no_secrets",
+  "CheckTitle": "Find secrets in EC2 Launch Template",
+  "CheckType": [],
+  "ServiceName": "ec2",
+  "SubServiceName": "",
+  "ResourceIdTemplate": "arn:partition:service:region:account-id:",
+  "Severity": "critical",
+  "ResourceType": "AwsEc2LaunchTemplate",
+  "Description": "Find secrets in EC2 Launch Template",
+  "Risk": "The use of a hard-coded password increases the possibility of password guessing.  If hard-coded passwords are used, it is possible that malicious users gain access through the account in question.",
+  "RelatedUrl": "",
```

Comment:
```suggestion
  "RelatedUrl": "https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/ec2-launch-templates.html",
```

Reply:
I saw that this URL explicitly references Windows. I'm instead resolving this with the more generic `https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-launch-templates.html` URL in the next commit.